schema/google/showcase/v1beta1/sequence.proto

```diff
@@ -217,14 +217,19 @@ message AttemptSequenceRequest {
     (google.api.resource_reference).type = "showcase.googleapis.com/Sequence",
     (google.api.field_behavior) = REQUIRED
   ];
+
 }
 
 message AttemptStreamingSequenceRequest {
   string name = 1 [
     (google.api.resource_reference).type = "showcase.googleapis.com/StreamingSequence",
     (google.api.field_behavior) = REQUIRED
   ];
-
+  
+  int32 fail_index = 2 [
+    (google.api.resource_reference).type = "showcase.googleapis.com/Sequence",
```

Comment:
@noahdietz Is this is a valid resource reference? I think the field reference another resource should be of String type per [AIP-122](https://google.aip.dev/122), also the [Sequence](https://github.com/googleapis/gapic-showcase/blob/fe2fe253b030604605ea926b7e4664585669969d/schema/google/showcase/v1beta1/sequence.proto#L92-L95) resource is of pattern `sequences/{sequence}` which seems do not fit here.

Reply:
Oh yeah definitely not.

Reply:
good catch! I accepted Noah's suggestion which iiuc just removed what used to be line 30 with this reference type. I added a clarifying comment, good call @blakeli0. I want to make sure I'm interpreting these comments and the code suggestion from @noahdietz correctly though - do I not need to include a resource type at all? Or, should I be linking to something like https://github.com/googleapis/gapic-showcase/blob/3188c8a6089597d937ed6ab5b529716ab92d5ff3/schema/google/showcase/v1beta1/sequence.proto#L233?

Reply:
Nope, not necessary. The resource here is the Sequence being attempted. The `content` is just data.